src/bindings/c/tests/ov_infer_request_test.cpp

```diff
@@ -8,20 +8,20 @@
 inline void get_tensor_info(ov_model_t* model, bool input, char** name, ov_shape_t* shape, ov_element_type_e* type) {
     ov_output_const_port* port = nullptr;
     if (input) {
-        OV_EXPECT_OK(ov_model_const_input(model, &port));
+        OV_ASSERT_OK(ov_model_const_input(model, &port));
```

Comment:
@riverlijunjie I think we shouldn't always use `ASSERT`.
`ASSERT` is needed if we want to finish test execution after the fail (for example after that we cannot work with incorrect object), but if we want just to check the value which doesn't impact future execution, it is better to use `EXPECT` in order to get all errors and don't stop on the first one.

Reply:
ok, let's check which cases need use `EXCEPT `and which need use `ASSERT`.